docker image build broken on missing python

### Description

When the deployment action runs `RUN npm ci --only=production && npm cache clean --force --loglevel=error` there is now a consistent failure

```
#8 16.92 npm ERR! path /home/node/app/node_modules/@parcel/watcher
#8 16.92 npm ERR! command failed
#8 16.92 npm ERR! command sh -c -- node-gyp-build
#8 16.92 npm ERR! gyp info it worked if it ends with ok
#8 16.92 npm ERR! gyp info using node-gyp@9.3.0
#8 16.92 npm ERR! gyp info using node@16.18.0 | linux | arm64
#8 16.92 npm ERR! gyp ERR! find Python
#8 16.92 npm ERR! gyp ERR! find Python Python is not set from command line or npm configuration
#8 16.92 npm ERR! gyp ERR! find Python Python is not set from environment variable PYTHON
#8 16.92 npm ERR! gyp ERR! find Python checking if "python3" can be used
#8 16.92 npm ERR! gyp ERR! find Python - "python3" is not in PATH or produced an error
#8 16.92 npm ERR! gyp ERR! find Python checking if "python" can be used
#8 16.92 npm ERR! gyp ERR! find Python - "python" is not in PATH or produced an error
#8 16.92 npm ERR! gyp ERR! find Python
#8 16.92 npm ERR! gyp ERR! find Python **********************************************************
#8 16.93 npm ERR! gyp ERR! find Python You need to install the latest version of Python.
#8 16.93 npm ERR! gyp ERR! find Python Node-gyp should be able to find and use Python. If not,
#8 16.93 npm ERR! gyp ERR! find Python you can try one of the following options:
#8 16.93 npm ERR! gyp ERR! find Python - Use the switch --python="/path/to/pythonexecutable"
#8 16.93 npm ERR! gyp ERR! find Python   (accepted by both node-gyp and npm)
#8 16.93 npm ERR! gyp ERR! find Python - Set the environment variable PYTHON
#8 16.93 npm ERR! gyp ERR! find Python - Set the npm configuration variable python:
#8 16.93 npm ERR! gyp ERR! find Python   npm config set python "/path/to/pythonexecutable"
#8 16.93 npm ERR! gyp ERR! find Python For more information consult the documentation at:
#8 16.93 npm ERR! gyp ERR! find Python https://github.com/nodejs/node-gyp#installation
#8 16.93 npm ERR! gyp ERR! find Python **********************************************************
#8 16.93 npm ERR! gyp ERR! find Python
#8 16.93 npm ERR! gyp ERR! configure error
#8 16.93 npm ERR! gyp ERR! stack Error: Could not find any Python installation to use
#8 16.93 npm ERR! gyp ERR! stack     at PythonFinder.fail (/home/node/app/node_modules/node-gyp/lib/find-python.js:330:47)
#8 16.93 npm ERR! gyp ERR! stack     at PythonFinder.runChecks (/home/node/app/node_modules/node-gyp/lib/find-python.js:159:21)
#8 16.93 npm ERR! gyp ERR! stack     at PythonFinder.<anonymous> (/home/node/app/node_modules/node-gyp/lib/find-python.js:202:16)
#8 16.93 npm ERR! gyp ERR! stack     at PythonFinder.execFileCallback (/home/node/app/node_modules/node-gyp/lib/find-python.js:294:16)
#8 16.93 npm ERR! gyp ERR! stack     at exithandler (node:child_process:410:5)
#8 16.93 npm ERR! gyp ERR! stack     at ChildProcess.errorhandler (node:child_process:422:5)
#8 16.93 npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
#8 16.93 npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)
#8 16.93 npm ERR! gyp ERR! stack     at onErrorNT (node:internal/child_process:485:16)
#8 16.93 npm ERR! gyp ERR! stack     at processTicksAndRejections (node:internal/process/task_queues:83:21)
#8 16.93 npm ERR! gyp ERR! System Linux 5.10.124-linuxkit
#8 16.93 npm ERR! gyp ERR! command "/usr/local/bin/node" "/home/node/app/node_modules/.bin/node-gyp" "rebuild"
#8 16.93 npm ERR! gyp ERR! cwd /home/node/app/node_modules/@parcel/watcher
#8 16.93 npm ERR! gyp ERR! node -v v16.18.0
#8 16.93 npm ERR! gyp ERR! node-gyp -v v9.3.0
#8 16.93 npm ERR! gyp ERR! not ok
```

### Steps to reproduce

build docker file with `docker build .`

### Additional context

_No response_

### Hedera network

other

### Version

v0.10.0

### Operating system

_No response_

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,5 @@
-FROM node:16.15-alpine3.15
+FROM node:16-alpine
+RUN apk update && apk add g++ make py3-pip
 
 # Setup
 ENV NODE_ENV production

